sklearn/ensemble/_voting.py

```diff
@@ -90,7 +90,36 @@ def fit(self, X, y, sample_weight=None):
             self.named_estimators_[name] = current_est
 
         return self
-
+    
+    def fit_transform(self, X, y=None, **fit_params):
+        """
+        Return class labels or probabilities for X for each estimator.
+        
+        Return predictions for X for each estimator.
+        
+        Parameters
+        ----------
+        X : {array-like, sparse matrix, dataframe} of shape \
+                (n_samples, n_features)
+            Input samples.
+        y : ndarray of shape (n_samples,), default=None
+            Target values (None for unsupervised transformations).
+        **fit_params : dict
+            Additional fit parameters.
+        Returns
+        -------
+        X_new : ndarray array of shape (n_samples, n_features_new)
+            Transformed array.
+        """
+        # non-optimized default implementation; override when a better
+        # method is possible for a given clustering algorithm
+        if y is None:
+            # fit method of arity 1 (unsupervised transformation)
+            return self.fit(X, **fit_params).transform(X)
+        else:
+            # fit method of arity 2 (supervised transformation)
+            return self.fit(X, y, **fit_params).transform(X)
+    
```

Comment:
you don't need this code. You just need to call `super`

Reply:
hey @glemaitre  I remove that block of code and added <code>super()</code> in second commit

have a look